tests/Zend/Db/Adapter/Pdo/SqliteTest.php

```diff
@@ -269,7 +269,7 @@ public function testAdapterQuoteNullByteCharacter()
     public function testAdapterZendConfigEmptyDriverOptions()
     {
         $params = $this->_util->getParams();
-        $params['driver_options'] = '';
+        $params['driver_options'] = null;
```

Comment:
Yep look logical to use []

> why `null` not keep empty string `''`?

To verify case use `Null Coalescing Operator` after apply your suggestion 

[library/Zend/Db/Adapter/Pdo/Sqlite.php](https://github.com/Shardj/zf1-future/pull/320/commits/633163b53142ba6c7e5539724183777bcf193426)

```diff
- $config['driver_options'] = empty($config['driver_options']) ? array() : $config['driver_options'];
+ $config['driver_options'] = $config['driver_options'] ?? [];
```

Reply:
`[]` would work for `??` as well.

Reply:
Yep, it's work

Reply:
Can you make the change then? :)

Reply:
> Can you make the change then? :)

it's done Mr @glensc 

Just for fun:
Before replace empty string `''` by `null` in test case, i predicted we would have a conversation about it, i am a prophet :)

Reply:
@hungtrinh that is good. no way can slip in extra hidden changes! you should have committed it separately with an explanation in the commit message body, then wouldn't have to discuss it in the review notes. i.e more transparent change. because it doesn't match the commit message it's committed under now.

Reply:
Mr @glensc please merge this PR

Reply:
I was hoping you split the commit. I guess not then.